app/src/main/java/uc/edu/ozturkkl/traveltracker/dao/LocationDAO.kt

```diff
@@ -1,9 +1,18 @@
 package uc.edu.ozturkkl.traveltracker.dao
 
-import uc.edu.ozturkkl.traveltracker.dto.Location
+import android.telecom.Call
+
 
 interface LocationDAO {
 
 
+    //fun getAllLocations():
```

Comment:
Don't commit commented out code to version control.  That's a no-no.  

1. You are in your own branch, you can do what you want without impacting master.  That's why this review is a separate branch.
2. You can always go back in history to see what used to be there.
3. You should only commit compiling code to master.   
4. In the instructions, I mentioned that you should make proposals in code; not have a discussion in code.